ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/textinput/v1/textinput/_cq_dialog/.content.xml

```diff
@@ -267,32 +267,33 @@
                                 <items jcr:primaryType="nt:unstructured">
                                     <validationPatternType
                                             jcr:primaryType="nt:unstructured"
+                                            granite:class="cmp-adaptiveform-textinput__validationpattern"
                                             sling:resourceType="granite/ui/components/coral/foundation/form/select"
                                             fieldLabel="Type"
-                                            name="./validationPatternType">
+                                            name="./validatePatternType">
+                                        <datasource
+                                                jcr:primaryType="nt:unstructured"
+                                                sling:resourceType="core/fd/components/form/container/v1/datasource"
+                                                fieldType="text-input"
+                                                type="formatters"/>
                                     </validationPatternType>
-                                    <validatePictureClause
+                                    <validateFormat
                                             jcr:primaryType="nt:unstructured"
+                                            granite:class="cmp-adaptiveform-textinput__validationformat"
                                             sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
                                             fieldDescription="Regular Expression is permitted."
                                             fieldLabel="Pattern"
-                                            name="./pattern">
-                                    </validatePictureClause>
+                                            name="./validateFormat">
```

Comment:
the name should be `pattern` as per spec. any reason why it's changed to validateFormat i the latest commit?

Reply:
this was a miss at my end, i was going through the migration spec to check what should be the name to save this field, i made it in the same format in displayformat,editFormat etc. Should have validated in Sling model.